Release managers asked for these changes to be postponed after etch,
to make sure that they do not break applications.

Index: glibc-2.3.6/localedata/locales/ru_RU
===================================================================
--- glibc-2.3.6.orig/localedata/locales/ru_RU
+++ glibc-2.3.6/localedata/locales/ru_RU
@@ -81,7 +81,7 @@
 int_curr_symbol           "<U0052><U0055><U0042><U0020>"
 currency_symbol           "<U0440><U0443><U0431>"
 mon_decimal_point         "<U002E>"
-mon_thousands_sep         "<U2002>"
+mon_thousands_sep         "<U0020>"
 mon_grouping              3;3
 positive_sign             ""
 negative_sign             "<U002D>"
@@ -97,7 +97,7 @@
 
 LC_NUMERIC
 decimal_point             "<U002C>"
-thousands_sep             "<U2002>"
+thousands_sep             "<U002E>"
 grouping                  3;3
 END LC_NUMERIC
 
Index: glibc-2.3.6/localedata/locales/vi_VN
===================================================================
--- glibc-2.3.6.orig/localedata/locales/vi_VN
+++ glibc-2.3.6/localedata/locales/vi_VN
@@ -235,8 +235,8 @@
 mon_grouping          3;3
 positive_sign         ""
 negative_sign         "<U002D>"
-int_frac_digits       0
-frac_digits           0
+int_frac_digits       4
+frac_digits           4
 p_cs_precedes         0
 p_sep_by_space        0
 n_cs_precedes         1
@@ -266,19 +266,19 @@
          "<U0054><U0068><U1EE9><U0020><U006E><U0103><U006D>";/
          "<U0054><U0068><U1EE9><U0020><U0073><U00E1><U0075>";/
          "<U0054><U0068><U1EE9><U0020><U0062><U1EA3><U0079>"
-abmon    "<U0054><U0068><U0030><U0031>";/
-         "<U0054><U0068><U0030><U0032>";/
-         "<U0054><U0068><U0030><U0033>";/
-         "<U0054><U0068><U0030><U0034>";/
-         "<U0054><U0068><U0030><U0035>";/
-         "<U0054><U0068><U0030><U0036>";/
-         "<U0054><U0068><U0030><U0037>";/
-         "<U0054><U0068><U0030><U0038>";/
-         "<U0054><U0068><U0030><U0039>";/
+abmon    "<U0054><U0068><U0031>";/
+         "<U0054><U0068><U0032>";/
+         "<U0054><U0068><U0033>";/
+         "<U0054><U0068><U0034>";/
+         "<U0054><U0068><U0035>";/
+         "<U0054><U0068><U0036>";/
+         "<U0054><U0068><U0037>";/
+         "<U0054><U0068><U0038>";/
+         "<U0054><U0068><U0039>";/
          "<U0054><U0068><U0031><U0030>";/
          "<U0054><U0068><U0031><U0031>";/
          "<U0054><U0068><U0031><U0032>"
-mon      "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U1ED9><U0074>";/
+mon      "<U0054><U0068><U00E1><U006E><U0067><U0020><U0067><U0069><U00EA><U006E><U0067>";/
          "<U0054><U0068><U00E1><U006E><U0067><U0020><U0068><U0061><U0069>";/
          "<U0054><U0068><U00E1><U006E><U0067><U0020><U0062><U0061>";/
          "<U0054><U0068><U00E1><U006E><U0067><U0020><U0074><U01B0>";/
@@ -288,12 +288,12 @@
          "<U0054><U0068><U00E1><U006E><U0067><U0020><U0074><U00E1><U006D>";/
          "<U0054><U0068><U00E1><U006E><U0067><U0020><U0063><U0068><U00ED><U006E>";/
          "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U1EDD><U0069>";/
-         "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U1EDD><U0069><U0020><U006D><U1ED9><U0074>";/
-         "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U01B0><U1EDD><U0069><U0020><U0068><U0061><U0069>"
-d_t_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U004E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
+         "<U0054><U0068><U00E1><U006E><U0067><U0020><U006D><U1ED9><U0074>";/
+         "<U0054><U0068><U00E1><U006E><U0067><U0020><U0063><U0068><U1EA1><U0070>"
+d_t_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U006E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt    "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0059>"
 t_fmt    "<U0025><U0054>"
-am_pm    "<U0041><U004D>";"<U0050><U004D>"
+am_pm    "<U0073><U00E1><U006E><U0067>";"<U0063><U0068><U0069><U1EC1><U0075>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U0020><U0025><U0070>"
 date_fmt  "<U0025><U0041><U002C><U0020><U0025><U0064><U0020><U0025><U0042><U0020><U006E><U0103><U006D><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 %  FIXME: found in CLDR
@@ -319,7 +319,7 @@
 % LC_NAME category.
 %
 name_fmt    "<U0025><U0070><U0025><U0074><U0025><U0066><U0025><U0074>/
-<U0025><U006D><U0025><U0074><U0025><U0067>"
+<U0025><U0067>"
 name_gen    ""
 name_mr     "<U00D4><U002E>"
 name_mrs    "<U0042><U002E>"
@@ -356,3 +356,4 @@
 LC_MEASUREMENT
 measurement 1
 END LC_MEASUREMENT
+
Index: glibc-2.3.6/localedata/ChangeLog
===================================================================
--- glibc-2.3.6.orig/localedata/ChangeLog
+++ glibc-2.3.6/localedata/ChangeLog
@@ -264,9 +264,6 @@
 	[BZ #388]
 	* locales/pl_PL: Don't ignore U0020 in collation.
 
-	[BZ #352]
-	* locales/ru_RU: Use U2002 for thousands_sep and mon_throusands_sep.
-
 2006-05-01  Alexander Shopov  <ash@contact.bg>
 
 	[BZ #2625]
@@ -305,10 +302,6 @@
 	* locales/pa_PK: New file.
 	* SUPPORTED: Add pa_PK.UTF-8.
 
-	[BZ #1974]
-	* locales/vi_VN: Fix int_frac_digits/frac_digits, abmon, mon, d_t_fmt,
-	am_pm, and name_fmt.
-
 	[BZ #1787]
 	* locales/en_ZA: Updates.
 
